extensions/wikia/AdEngine/AdEngine2ContextService.class.php

```diff
@@ -24,6 +24,11 @@ public function getContext( Title $title, $skinName ) {
 				$sevenOneMediaCombinedUrl = ResourceLoader::makeCustomURL( $wg->Out, ['wikia.ext.adengine.sevenonemedia'], 'scripts' );
 			}
 
+			$monetizationServiceAds = null;
+			if ( !empty( $wg->AdDriverUseMonetizationService ) && !empty( $wg->EnableMonetizationModuleExt ) ) {
+				$monetizationServiceAds = F::app()->sendRequest( 'MonetizationModule', 'index' )->getData()['data'];
```

Comment:
[Errors](http://graph-s3/grafana/#/dashboard/db/Monetization%20Service?panelId=5&fullscreen&from=now-24h&to=now)?  :smiling_imp: